### PR TITLE
mysql-test, part 15

### PR DESCRIPTION
**Pull request by osheroff**
MERGE MANUALLY

- handle table names and ASC/DESC in `ALTER TABLE ORDER BY` ddl

@zendesk/rules

### CODE REVIEW

File: src/main/antlr4/imports/mysql_alter_table.g4 [hunk 1]
```diff
@@ -65,3 +65,9 @@ ignored_alter_specifications:
     ;
   algorithm_type: DEFAULT | INPLACE | COPY;
   lock_type: DEFAULT | NONE | SHARED | EXCLUSIVE;
+
+alter_ordering: alter_ordering_column (ASC|DESC)?;
+alter_ordering_column:
+    name '.' name '.' name
```
Comment (dadah89) [inline src/main/antlr4/imports/mysql_alter_table.g4]: would `table_name '.' name | name;` work here? does it matter adding a test for that case?

**Comment by dadah89** [comment]
:+1:

File: src/main/antlr4/imports/mysql_alter_table.g4 (hunk 1 above)
Comment (osheroff) [inline src/main/antlr4/imports/mysql_alter_table.g4]: not quite, there's three forms of column names that are *technically* acceptable:

```
`col`
`tbl`.`col`
`db`.`col`
```

In this case, it doesn't matter if I give them aliased names, as I'm parsing and throwing away all of this stuff.

Comment (dadah89) [inline src/main/antlr4/imports/mysql_alter_table.g4]: thought that table_name would take care of tbl and db.tbl, putting the logic in one place... think it's a small thing though, pr looks just fine to me

**Comment by osheroff** [comment]
merged.

**Pull request closed without merging**